src/app/frontend/chrome/chrome.html

```diff
@@ -17,12 +17,14 @@
 <md-toolbar class="kd-toolbar">
   <div class="md-toolbar-tools kd-toolbar-tools">
     <div class="kd-logo-bar">
-      <md-icon md-svg-icon="assets/images/kubernetes-logo.svg"
-               class="kd-toolbar-logo">
-      </md-icon>
-      <md-icon md-svg-icon="assets/images/kubernetes-logo-text.svg"
-               class="kd-toolbar-logo-text">
-      </md-icon>
+      <a ui-sref="{{$ctrl.overviewState()}}" class="kd-toolbar-logo-link">
```

Comment:
1. `ui-sref` does not have to be placed on `a` tag. Move it to the `div` above and remove `a`.
2. Class `kd-toolbar-logo-link` does not exist. You have to add it to `chrome.scss`.

Reply:
1. `ui-sref` binds `a` tag to as far as I know. I used [this](https://github.com/angular-ui/ui-router/wiki/Quick-Reference#ui-sref) for reference
2. Will do it once you confirm the above

Reply:
I have checked and it also works on other tags. Only difference is that you need to style it manually.

Reply:
Yeah. But it makes the whole `kd-logo-bar` clickable. I mean the empty space between the logo text and search bar. We could apply some style to prevent that. But do you want the whole `kd-logo-bar` to be clickable?

Reply:
I think it's ok. It is considered as a single element.

Reply:
Ahh, this element is very wide. Then it's ok to add extra tag to wrap icons.